### PR TITLE
Fix serialization doc display

### PR DESCRIPTION
**Pull request by jerryzh168**
Summary:
att

Test Plan:
checkout rendered doc

Reviewers:

Subscribers:

Tasks:

Tags:

### CODE REVIEW

**Comment by msaroufim** [comment]
When you're ready just please post a screenshot to the page and we should be good to merge

**Comment by jerryzh168** [comment]
Screenshot 2024-07-19 at 10 54 09

formatting looks good now: https://docs-preview.pytorch.org/pytorch/ao/525/serialization.html

**Review by msaroufim — APPROVED** [review]

File: docs/source/index.rst [hunk 1]
```diff
@@ -97,5 +96,7 @@ with more content coming soon.
    api_ref_intro
    api_ref_quantization
    api_ref_dtypes
+   serialization
+
```
Comment (msaroufim) [inline docs/source/index.rst:100]: you need to create a second caption for docs cause tutorial is not an API reference

Comment (jerryzh168) [reply docs/source/index.rst:100]: this is also not a tutorial, should I add "Concepts"?

**Pull request merged**